packages/components/src/ComboBox/src/ComboBox.tsx

```diff
@@ -77,20 +89,30 @@ function ComboBox<T extends object>(props: ComboBoxProps<T>, ref: ForwardedRef<H
     const { stylingProps, ...ownProps } = useStyledSystem(props);
     const {
         className,
-        children: childrenProp,
+        children,
+        description,
+        errorMessage,
+        footer,
+        isAutoMenuWidth,
         isFluid: isFluidProp,
         isInvalid,
         isRequired,
         items,
+        label,
+        listBoxProps,
         menuTrigger = "focus",
         necessityIndicator,
+        popoverProps = {
```

Comment:
could we expose a property menuDirection instead?

https://react-spectrum.adobe.com/react-spectrum/ComboBox.html#menu-direction

Reply:
Just confirming because they actually just name it "direction" but you think it should be "menuDirection"?

Reply:
direction is fine by me